docs/vault.md

```diff
@@ -81,6 +81,13 @@ vault-radar scan-confluence -outfile=confluence.csv -url="http://localhost:8090"
 An in-memory index keyed off of secret hashes will be generated prior to scanning the source. This index will be used to
 annotate whether a risk exists in Vault.
 
+### Generate index file using HCP provided salt
+Index files are generated in an "online mode" by default, meaning that the secret hash produced is using a salt that is provided from HCP. This requires the Project Service Principals to be configured for your system as outlined by the [HCP Upload Section](hcp-upload.md). If you do not want to use the index file for HCP upload and visualization you can use the `--offline` flag to use a local hashing salt and not error if the Service Principals are not configured.
+
+```bash
+vault-radar scan-aws-parameter-store index -r <REGION CODE> -o <PATH TO OUTPUT>.jsonl --offline
```

Comment:
the example is a little bit confusing as it will use the "offline" mode, while the title of the section is about HCP/online mode. Maybe there should be two examples for both online and offline modes and maybe a separate section for offline mode...